drivers/at86rf215/at86rf215_netdev.c

```diff
@@ -52,6 +52,21 @@ const netdev_driver_t at86rf215_driver = {
     .set = _set,
 };
 
+static bool _is_busy(at86rf215_t *dev)
+{
+    if (dev->flags & AT86RF215_OPT_TX_PENDING) {
+        return true;
+    }
+
+    if (dev->state == AT86RF215_STATE_TX ||
+        dev->state == AT86RF215_STATE_TX_WAIT_ACK ||
+        dev->state == AT86RF215_STATE_RX_SEND_ACK) {
+        return true;
+    }
+
+    return false;
+}
```

Comment:
Can you put some code in common with `_tx_ongoing`?

Reply:
I thought about that too, but you can still fill the TX buffer (and even queue TX) while in `AT86RF215_STATE_RX_SEND_ACK`.

I still hope we can eventually get rid of that `_block_while_busy()` loop if something like #11263 gets merged.

Reply:
Yes I understand that, but it still seems to be that you could still have:

```
static bool _is_busy(at86rf215_t *dev)
{
    if (_tx_ongoing() ||  dev->state == AT86RF215_STATE_RX_SEND_ACK) {
        return true;
    }

    return false;
}
```

Reply:
It's in two different files so it would need to be a global function… (yea the split between `at86rf215.c` and `at86rf215_netdev.c` is completely arbitrary, they should probably be merged.

Reply:
Ohhh... got it, well then I'm Ok with leaving it as is.

Reply:
> I thought about that too, but you can still fill the TX buffer (and even queue TX) while in AT86RF215_STATE_RX_SEND_ACK.


Actually can you add that as a comment. You can squash and trigger murdock after.

Reply:
I added a comment to `_tx_ongoing()`